tests/appsec/waf/test_miscs.py

```diff
@@ -79,11 +79,12 @@ def test_main(self):
 
 @missing_feature(library="golang")
 @missing_feature(library="dotnet")
-@missing_feature(library="nodejs")
 @missing_feature(library="java")
 @missing_feature(library="php")
 @missing_feature(library="python")
 @missing_feature(library="ruby")
+@released(nodejs="2.4.0")
+@irrelevant(reason="test not implemented")
```

Comment:
Even if it's the test that is not yet done, we flag it as missing feature. 

```suggestion
@missing_feature(reason="test not implemented")
```